src/cursor/cur_backup.c

```diff
@@ -650,18 +648,15 @@ __backup_config(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, const char *cfg[
 
         /*
          * Handle log targets. We do not need to go through the schema worker, just call the
-         * function to append them. Set log_only only if it is our only URI target.
+         * function to append them.
          */
         if (WT_PREFIX_MATCH(uri, "log:")) {
```

Comment:
I am not clear if this goes away with log-based backup. We do not have an example program for Target backups.

Reply:
That targeted backup feature is orthogonal to incremental backup - it's a full backup of specific tables (or whatever you put in the URI, I guess). This particular line is still needed for extending backup cursors.

Reply:
I suggest enhancing WiredTiger > Reference Guide > Writing WiredTiger applications > Backups > Targeted backups in wiredtiger/src/docs/arch-backup.dox. We don't say whether a Target backup is full or incremental, and whether it is log-based or block-based. I suggest adding that it is full.

It currently says:

# Target backup #

When the backup cursor is opened with the \c target configuration string as \c "target=",
WiredTiger internally generates a list of targeted files instead of all files like the full backup.
If the targeted object is a table, all the indexes and column groups of the table are also backed up.

Note: There can only be one backup cursor open at a time unless using duplicate backup
cursors. The duplicate backup cursors are used for catching up with the log files or with
block-based incremental backup. Please refer to @ref backup_duplicate for more details.

I suggest changing it to:
A target backup is a full backup of only the specified targets. The rest the same as before.

Reply:
I suggest an improvement to the documentation of duplicate backup cursors. It is not clear which types of backups can use duplicate backup cursors.  Duplicate backup cursors are documented only in WiredTiger > Reference Guide > Writing WiredTiger applications > Backups > Duplicate backup cursors in wiredtiger/src/docs.backup.dox. However they are mentioned in WiredTiger > Reference Guide > WiredTiger Architecture Guide > On Disk Concepts > Backup > Target Backup. It says "The duplicate backup cursors are used for catching up with the log files or with block-based incremental backup." From this I conclude duplicate backup cursors can be used with full backups, log-based incremental backups, and block-based incremental backups. This should be added to WiredTiger > Reference Guide > Writing WiredTiger applications > Backups > Duplicate backup cursors in wiredtiger/src/docs.backup.dox.

Reply:
Given that we have a ticket for removing the log-based incremental backup documentation (WT-12471), can I ask that you post the comments there?